### ISSUE TITLE
add new value "CREATING" to \SellingPartnerApi\Model\FbaInbound\ShipmentStatus

### ISSUE THREAD

**Issue by suofiya** [jlevers/selling-partner-api#258]
There is status "CREATING" that is missing from Amazon's documentation and I get an error if there is a shipment with this status returned by SellingPartnerApi\Api\FbaInboundApi->getShipments();

Can you add a constant in \SellingPartnerApi\Model\FbaInbound\ShipmentStatus to fix the issue.

```
{"shipment_data":[{"ShipmentId":"xxxxxxxxxxx","ShipmentName":"STAR Reconciliation","ShipFromAddress":[],"DestinationFulfillmentCenterId":"YYYY","ShipmentStatus":"CREATING","LabelPrepType":"NO_LABEL"}]}
```

**Comment by suofiya** [comment]
Another new Status is 'ABANDONED'.

**Comment by jlevers** [comment]
I'm releasing `v5.0` soon -- this will be fixed in that release. I'll update this issue when I push the new version.

**Comment by jlevers** [comment]
This should be fixed in `v5.0.0`. Give it a try and let me know if you're still having issues!
